clean-sensitive-data/main.bal

```diff
@@ -0,0 +1,24 @@
+import ballerina/io;
+
+public function removeEmailStrings(string inputString) returns string {
+    string:RegExp emailPattern =  re `[A-Za-z0-9\._%+-]+@[A-Za-z0-9\.-]+\.[A-Za-z]{2,}`;
+    return emailPattern.replaceAll(inputString, "");
+}
+
+public function removeSSN(string inputString) returns string {
+    string:RegExp ssnPattern = re `\d{3}-\d{2}-\d{4}`;
+    return ssnPattern.replaceAll(inputString, "");
+}
+
+public function removeExtraWhiteSpaces(string inputString) returns string {
+    string:RegExp extraWhiteSpacesPattern = re `\s+`;
+    return extraWhiteSpacesPattern.replaceAll(inputString, " ");
+}
+
+public function main() {
+    string inputString = "";
+    inputString = removeEmailStrings(inputString);
```

Comment:
shall we rename this variable, because name like inputString is not make sense in here.